docs/bigquery/snippets.py

```diff
@@ -2162,7 +2162,7 @@ def test_undelete_table(client, to_delete):
     # to snapshot a table prior to its creation time is an error.
     table = client.get_table(table_ref)
     if table.created_time > snapshot_time:
-      snapshot_time = table.created_time
+        snapshot_time = table.created_time
```

Comment:
Please revert:  this is a stray change, unrelated to the PR.  I've just merged a fix to `master` in PR #5733, which you can pick up via a rebase.

Reply:
Thank you, I just did it for the sake of passing the checks. removed the commit and rebased.